### ISSUE TITLE
🌟 [EPIC] Version BabyVersion

### ISSUE THREAD

**Issue by Elweyn** [dreammall-earth/dreammall.earth#1082]
## 🌟 EPIC
- Als Gast kann ich die presentations Webseite https://dreammall.earth/ sehen.
- Als Gast kann ich von der presentation mich in die DreamMall Application https://app.dreammall.earth anmelden.
  - Als Gast kann ich mir ein Account auf Authentik erstellen
  - Als Nutzer kann ich mich über Authentik anmelden
- Als Nutzer kann ich das Sternenbild (Weltencafé) sehen
  - Als Nutzer kann ich einem Raum beitreten
- Als Nutzer kann ich das Cockpit über den TabControl erreichen
- Als Nutzer kann ich die öffentlichen Räumen über den Header sehen

## 🤖 ToDos:
- [ ] #2 
- [ ] #545 
- [ ] #284 
- [ ] #290 
- [ ] #145 

## Baby Version:
- Presenter
- Authentik
- Frontend
  - Header
    - TabControl
    - Switch Light/Dark
    - Offene Räume
    - UserButton
      - Settings
      - Sign Out
  - Sternenbild (Weltencafé)
    - Knopf
      - Mein Raum
    - Offene Räume
  - Cockpit
    - Über mich
    - Meine Räume
    - Meine Gruppen
    - Upgrade (Mall Ticket)
- Admin Bereich
  - Übersicht der Nutzer
  - Übersicht der Räume

**Comment by Tirokk** [comment]
Der Punkt Einladung fehlt in meinen Augen noch.

**Comment by Elweyn** [comment]
> Benachrichtigungen sind also nicht dabei, oder?

Genau. Benachrichtigungen werden dann relevant wenn die Projekte implementiert sind

**Comment by Bettelstab** [comment]
Hmmm. Es gibt im Frontend ein Issue für die Gruppenübersicht, aber keines, um die Gruppe zu erstellen. Nach den Designvorlagen ist das ein sehr umfangreicher Punkt.
